src/components/Header/Header.tsx

```diff
@@ -44,134 +46,32 @@ export const Header = chakra(() => {
-                  <Icon name="arrow" align="right" size="small" />
-                </Button>,
-                <Link href="#" key="locationsLink">
-                  Locations
-                </Link>,
-                <Link href="#" key="libraryCardLink">
-                  Get A Library Card
-                </Link>,
-                <Button
-                  buttonType="link"
-                  className="withIcon"
-                  id="emailUpdatesButton"
-                  key="emailUpdatesButton"
-                >
-                  Get Email Updates
-                  <Icon name="arrow" align="right" size="small" />
-                </Button>,
-                <Button
-                  buttonType="callout"
-                  id="donateButton"
-                  key="donateButton"
-                >
-                  Donate
-                </Button>,
-                <Link href="#" key="shopLink">
-                  Shop
-                </Link>,
-              ]}
-              noStyling
-              type="ul"
-            />
-            <List
-              __css={styles.lowerLinks}
-              id="header-nav-lower"
-              inline
-              listItems={[
-                <Link href="#" key="booksMusicMoviesLink">
-                  Books/Music/Movies
-                </Link>,
-                <Link href="#" key="researchLink">
-                  Research
-                </Link>,
-                <Link href="#" key="educationLink">
-                  Education
-                </Link>,
-                <Link href="#" key="eventsLink">
-                  Events
-                </Link>,
-                <Link href="#" key="connectLink">
-                  Connect
-                </Link>,
-                <Link href="#" key="giveLink">
-                  Give
-                </Link>,
-                <Link href="#" key="getHelpLink">
-                  Get Help
-                </Link>,
-                <Button buttonType="link" id="searchButton" key="searchButton">
-                  Search
-                  <Icon name="search" align="right" size="small" />
-                </Button>,
-              ]}
-              noStyling
-              type="ul"
-            />
-          </VStack>
-        </Flex>
-      ) : (
-        <Flex>
-          <Box>
+        <HStack
+          marginX={windowDimensions.width > breakpointLarge ? "50px" : "0px"}
+          justifyContent="space-between"
+        >
+          <Link href="/" __css={styles.logo}>
             <Logo
-              __css={{ ...styles.logo, marginTop: "13px", height: "30px" }}
               id="header-nypl-logo"
-              name="nyplFullBlack"
-              size="xxsmall"
+              name={
+                windowDimensions.width > breakpointLarge
+                  ? "nyplFullBlack"
+                  : "nyplLionBlack"
+              }
+              size={
+                windowDimensions.width > breakpointLarge ? "small" : "xxsmall"
```

Comment:
I might be misunderstanding this comment, but here I'm trying to say if the window with is bigger than breakpointLarge, render size small, if window is smaller than breakpointLarge but bigger than breakpointMedium, render size xxsmall, if it's smaller than breakpointMedium, render Mobile.

Reply:
Yeah, your explanation makes sense but what I was trying to say is that the Logo here wasn't rendered but instead the Logo in the Mobile component was being rendered for small widths. So instead of having to render two different Logo components in two separation locations that depend on the size of the window, it is better to render Logo once and update it's size. Now this is fixed because Logo is being rendered once in `Header` and no longer in `Mobile`.